pkg/edit/store_api.go

```diff
@@ -76,7 +76,7 @@ func dedupCmds(allCmds []store.Cmd, newestFirst bool) []store.Cmd {
 	// Capacity allocation below is based on some personal empirical observation.
 	uniqCmds := make([]store.Cmd, 0, len(allCmds)/4)
 	seenCmds := make(map[string]bool, len(allCmds)/4)
-	for i := len(allCmds) - 1; i != 0; i-- {
+	for i := len(allCmds) - 1; i >= 0; i-- {
```

Comment:
Great catch of a classic off-by-one bug. I'm doing a "Picard face-palm" at myself 😄 This is also a good example of a fencepost error (a special-case of the off-by-one error).

Reply:
Yes indeed, happens to us all ;)